### DISCUSSION TITLE
Deleting a user

### DISCUSSION THREAD

**Discussion by akshaykumarp12** [pocketbase/pocketbase#4819] [Q&A]
there is  option to delete account for the user in flutter app. when deleting account password is prompted from user. how to validate the user with this password to delete account.

**Comment by akkisagiraju** [comment]
I think it should be possible to authenticate the user again before running the delete operation.

This documentation should help you: https://pocketbase.io/docs/api-records/#auth-with-password

```dart
final authData = await pb.collection('users').authWithPassword(
  'YOUR_USERNAME_OR_EMAIL',
  'YOUR_PASSWORD',
);
```

If the authStore is valid, you can proceed to delete the user.
